skipper_test.go

```diff
@@ -218,92 +216,67 @@ func TestHTTPServer(t *testing.T) {
 }
 
 func TestHTTPServerShutdown(t *testing.T) {
-	d := 1 * time.Second
+	o := &Options{}
+	testServerShutdown(t, o, "http")
+}
 
-	o := Options{
-		Address:                    ":19999",
-		WaitForHealthcheckInterval: d,
+func TestHTTPSServerShutdown(t *testing.T) {
+	o := &Options{
+		CertPathTLS: "fixtures/test.crt",
+		KeyPathTLS:  "fixtures/test.key",
 	}
+	testServerShutdown(t, o, "https")
+}
+
+func testServerShutdown(t *testing.T, o *Options, scheme string) {
+	const shutdownDelay = 1 * time.Second
```

Comment:
this is a `d` if you are writing Go :) 
If you write Java you can name it as in Java :)

Reply:
In java that would be [`HasThisTypePatternTriedToSneakInSomeGenericOrParameterizedTypePatternMatchingStuffAnywhereVisitor`](https://github.com/eclipse/org.aspectj/blob/master/org.aspectj.matcher/src/main/java/org/aspectj/weaver/patterns/HasThisTypePatternTriedToSneakInSomeGenericOrParameterizedTypePatternMatchingStuffAnywhereVisitor.java) :point_up: